### PR TITLE
core: frontend: fix SemVer not being populated

### LINKED ISSUE #1736: FirmwareInfo.Version is a string instead of the declared SemVer object
image
image

### PR DESCRIPTION
**Pull request by Williangalvani**
fix #1736

### CODE REVIEW

**Review by joaoantoniocardoso — APPROVED** [review]

**Review by joaoantoniocardoso — CHANGES_REQUESTED** [review]
wait... haha let me see it again

File: core/frontend/src/components/autopilot/AutopilotManagerUpdater.ts [hunk 1]
```diff
@@ -74,6 +75,8 @@ export async function fetchFirmwareInfo(): Promise<void> {
       url: `${autopilot.API_URL}/firmware_info`,
       timeout: 10000,
     })
+    // Version comes out as a string, let's turn it into a SemVer object
+    response.data.version = new SemVer(response.data.version)
```
Comment (joaoantoniocardoso) [inline core/frontend/src/components/autopilot/AutopilotManagerUpdater.ts:79]: Is this the only place we need to make this conversion?

Comment (Williangalvani) [reply core/frontend/src/components/autopilot/AutopilotManagerUpdater.ts:79]: it is the only one I found so far. I'm too deep in the params thing to look for others...

Comment (joaoantoniocardoso) [reply core/frontend/src/components/autopilot/AutopilotManagerUpdater.ts:79]: I checked, and it is.

**Review by joaoantoniocardoso — APPROVED** [review]

**Pull request merged**